src/components/ChangePlan/PlanCalculator.js

```diff
@@ -343,6 +344,11 @@ const PlanCalculator = ({ location, dependencies: { planService, appSessionRef }
     fetchData();
   }, [actionTypes.INIT, appSessionRef, planService, planType, pathType]);
 
+  useEffect(() => {
+    setActiveClass('');
+    setTimeout(() => setActiveClass('active'));
+  }, [planType]);
+
```

Comment:
You can you useTimeout() hook. 
PR https://github.com/FromDoppler/doppler-webapp/pull/1428
this to avoid introducing warnings in the console and in the tests about unsubscription

Reply:
I tried, but I had some troubles to use hooks inside a hook. Maybe I'm not using it well 🤔

Reply:
Well... it's not so bad. I should do something like: 
```
useEffect(() => {
    setActiveClass('');
    createTimeout(() => setActiveClass('active'));
  }, [planType, createTimeout]);
```

I'm not convinced adding createTimeout as a parameter in useEffect hook, but it works.

Reply:
you would be using a function (createTimeout) inside a hook. The hook would be `useTimeout ()`

Reply:
similar to how useState is used, but instead of returning this:

`const [myState, setMyStet] = useState ()`

it would return this:

`const createTimeout = useTimeout ();`